README.md

```diff
@@ -0,0 +1,198 @@
+# GCP Dataflow Example Project
+
```

Comment:
Any [badges](https://github.com/snowplow/schema-guru/blame/master/README.md#L3)? This is optional I think.